documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -37,28 +37,40 @@ When using Kafka with a Kafka cluster not managed by Strimzi, you can specify th
 [id='con-common-configuration-ssl-{context}']
 === `ssl`
 
-Use the three allowed `ssl` configuration options for client connection using a specific _cipher suite_ for a TLS version.
-A cipher suite combines algorithms for secure connection and data transfer.
+You can incorporate SSL configuration and cipher suite specifications to further secure TLS-based communication between your client application and a Kafka cluster.
+In addition to the standard TLS configuration, you can specify a supported TLS version and enable cipher suites in the configuration for the Kafka broker.
+You can also add the configuration to your clients if you wish to limit the TLS versions and cipher suites they use. 
+The configuration on the client must only use protocols and cipher suites that are enabled on the broker. 
 
-You can also configure the `ssl.endpoint.identification.algorithm` property to enable or disable hostname verification.
+A cipher suite is a set of algorithms for secure connection and data transfer.
+For example, a cipher suite specification might use a combination of the following algorithms:
+
+* TLS authentication
```

Comment:
I am not sure we can say the TLS authentication is an algorithm and it's part of a cipher suite ...

Reply:
changed to:

> For example, the cipher suite 'TLS_AES_256_GCM_SHA384' is composed of the following mechanisms, which are used in conjunction with the TLS protocol:

Reply:
Much better thanks!